docs/pages/pmd/devdocs/writing_documentation.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Writing Documentation
+last_update: August 2017
+permalink: pmd_devdocs_writing_documentation.html
+keywords: documentation, jekyll, markdown
+---
+
+PMD's documentation uses [Jekyll](https://jekyllrb.com/) with
+the [I'd rather be writing Jekyll Theme](http://idratherbewriting.com/documentation-theme-jekyll/index.html).
+
+Here are some quick tips.
+
+## Format
+
+The pages are in general in [Github Flavored Markdown](https://kramdown.gettalong.org/parser/gfm.html).
+
+## Structure
+
+All documentation is stored in the folder `docs/`. This is the folder, that github and the travis-ci scripts
+use to render the site.
+
+New pages are stored in the different subfolders under `pages`. The folder structure resembles the sidebar structure.
+Since all pages use a simple *permalink*, in the rendered html pages, all pages are flattened in one directory.
+This makes it easy to view the documentation also offline.
+
+## Building
+
+There are two ways, to execute jekyll:
+
+1.  Using [bundler](http://bundler.io/). This will install all the needed ruby packages locally and execute jekyll:
+
+        # this is required only once, to download and install the dependencies
+        bundle install
+        # this builds the documentation under _site
+        bundle exec jekyll build
+        # this runs a local webserver as http://localhost:4005
+        bundle exec jekyll serve
+
```

Comment:
Sidenote, I shared the script I use to start the local server as [a gist](https://gist.github.com/oowekyala/ee6f8801138861072c59ce683bdf737b)

Feel free to link to it if you think it has its place there